src/mokernels/mokernel.jl

```diff
@@ -4,3 +4,31 @@
 Abstract type for kernels with multiple outpus.
 """
 abstract type MOKernel <: Kernel end
+
+"""
+    matrixkernel(k::MOKernel, x, y)
```

Comment:
```suggestion
    matrixkernel(k::MOKernel, x, y[, out_dim])
```
& explanation below?